### ISSUE TITLE
Allow to overwrite default API response

### ISSUE THREAD

**Issue by esscue23** [springdoc/springdoc-openapi#35]
In my controller, I'm forced to use a custom serializer for some of my response objects. The implementation looks similar to this example here:
```Java	
        @RequestMapping(method = RequestMethod.GET)
	@ApiResponse(responseCode = "200", description = "Returns special things.", 
	content = @Content(
			schema = @Schema(
					type = "object",
					implementation = DummyClassForClassWithCustomSerializer.class
			)))
	public ResponseEntity<ClassWithCustomSerializer> doGet() {
        ...
        }
```

`ClassWithCustomSerializer` is serialized in a custom way. Springdoc OpenAPI will generate documentation based on the default serialization, which obviously doesn't match. I tried to add a dedicated @ApiResponse including a dummy class `DummyClassForClassWithCustomSerializer` corresponding to the custom serialization. Unfortunately, Springdoc OpenAPI ignores this annotation.

Would it be possible to allow to overwrite the default response object type by adding a specific @ApiResponse annotation for responseCode 200?

**Comment by springdoc** [comment]
Hi, 

  You can use: @ApiResponses in order to override the responses.
  It's more convenient to use @ApiResponse if you want to override only one return. 
  This feature will be available with v1.1.21 release.

**Comment by esscue23** [comment]
Thank you for the quick fix! I can confirm it's working with v1.1.21.
